Mage/src/main/java/mage/game/GameState.java

```diff
@@ -699,6 +699,11 @@ public void removeEotEffects(Game game) {
         game.applyEffects();
     }
 
+    public void removeTurnStartEffect(Game game) {
+        delayed.removeStartOfNewTurn(game);
```

Comment:
Of note, we may want to make the same for continuous effects "Until your next turn", instead of the current way?